lib/castle_web/search.ex

```diff
@@ -0,0 +1,47 @@
+defmodule CastleWeb.Search do
+
+  import Ecto.Query
+
+  def prefix_search(query) do
```

Comment:
Getting an error with an empty `?search=`.  Maybe catch that with a `def prefix_search("") do` or something?

Reply:
Done and added a test and a guard (on the off chance we can catch future bugs with the compiler)